frontend/src/components/MapView/Layers/LayerDropdown.tsx

```diff
@@ -16,16 +18,41 @@ import {
 import { useSafeTranslation } from '../../../i18n';
 import { getLayerGeometryIcon } from './layer-utils';
 
+const useStyles = makeStyles(() =>
+  createStyles({
+    selectRoot: {
+      color: 'black',
+      minWidth: '400px',
+      maxWidth: '450px',
+      width: 'auto',
+      '& label': {
+        color: '#333333',
+      },
+      '& .MuiInputBase-root': {
```

Comment:
Should this be handled by passing another class to InputProps?
Not a blocker but seeing many specific `.Mui`  cases is often an orange flag